app/controllers/proposal_controller.rb

```diff
@@ -24,15 +24,17 @@ def new
   end
 
   def edit
-    authorize! :edit, @event
```

Comment:
We already have following line on [top](https://github.com/sonalkr132/osem/blob/proposal-test/app/controllers/proposal_controller.rb#L5):
```Ruby
  load_and_authorize_resource :event, parent: false, through: :program, except: [:new, :create]
```
We don't need to authorize again.